ext/compressors/iaa/iaacodec/src/IAACompressionCodecDeflate.cpp

```diff
@@ -0,0 +1,257 @@
+#include <thread>
+#include <cstdio>
+#include "IAACompressionCodecDeflate.h"
+
+namespace DB
+{
+/* Local compressor structure. */
+typedef struct {
+    WT_COMPRESSOR compressor; /* Must come first */
+    WT_EXTENSION_API *wt_api; /* Extension API */
+} iaa_COMPRESSOR;
+
+static void
+iaa_message(WT_COMPRESSOR *compressor, WT_SESSION *session, const char *call, const char *info,  int msg)
```

Comment:
Extra space after the last comma:
```suggestion
iaa_message(WT_COMPRESSOR *compressor, WT_SESSION *session, const char *call, const char *info, int msg)
```

Reply:
Remove the extra space.